app/models/manageiq/providers/oracle_cloud/cloud_manager/cloud_database.rb

```diff
@@ -1,2 +1,167 @@
 class ManageIQ::Providers::OracleCloud::CloudManager::CloudDatabase < ::CloudDatabase
+  supports :create
+
+  def self.params_for_create(ems)
+    {
+      :fields => [
+        {
+          :component => 'text-field',
+          :id        => 'name',
+          :name      => 'name',
+          :label     => _('Cloud Database Name'),
+        },
+        {
+          :component    => 'select',
+          :name         => 'database',
+          :id           => 'database',
+          :label        => _('Database Type'),
+          :includeEmpty => true,
+          :isRequired   => true,
+          :options      => ['oracle', 'mysql'].map do |db|
+            {
+              :label => db,
+              :value => db
+            }
+          end,
+        },
+        {
+          :component    => 'select',
+          :name         => 'availability_domain',
+          :id           => 'availability_domain',
+          :label        => _('Availability Domain'),
+          :includeEmpty => true,
+          :isRequired   => true,
+          :options      => ems.availability_zones.map do |az|
+            {
+              :label => az[:name],
+              :value => az[:name]
+            }
+          end,
+        },
+        {
+          :component  => 'text-field',
+          :id         => 'compartment_id',
+          :name       => 'compartment_id',
+          :label      => _('Compartment ID'),
+          :isRequired => true,
+        },
+        {
+          :component  => 'text-field',
+          :id         => 'username',
+          :name       => 'username',
+          :label      => _('Admin Username'),
+          :isRequired => true,
+          :condition  => {
+            :when => 'database',
+            :is   => 'mysql',
+          },
+        },
+        {
+          :component  => 'text-field',
+          :id         => 'password',
+          :name       => 'password',
+          :label      => _('Admin Password'),
+          :isRequired => true,
+        },
+        {
+          :component  => 'text-field',
+          :id         => 'shape_name',
+          :name       => 'shape_name',
+          :label      => _('Database Shape Name'),
+          :isRequired => true,
+          :condition  => {
+            :when => 'database',
+            :is   => 'mysql',
+          },
+        },
+        {
+          :component    => 'select',
+          :name         => 'subnet_name',
+          :id           => 'subnet_name',
+          :label        => _('Cloud Subnet'),
+          :includeEmpty => true,
+          :isRequired   => true,
+          :condition    => {
+            :when => 'database',
+            :is   => 'mysql',
+          },
+          :options      => ems.cloud_subnets.map do |cs|
+            {
+              :label => cs[:name],
+              :value => cs[:name]
```

Comment:
We'd be better off using the id here, will make it much more reliable when querying later on
```suggestion
              :label => cs.name,
              :value => cs.id.to_s
```